docs/prerequisites.adoc

```diff
@@ -1,47 +1,35 @@
 = Pre-requisites
-
 :idprefix:
 :idseparator: -
 :sectlinks:
+:toc:
+:toc-placement!:
 
 :uri-repo: https://github.com/oracle-terraform-modules/terraform-oci-vcn
-
 :uri-rel-file-base: link:{uri-repo}/blob/master
 :uri-rel-tree-base: link:{uri-repo}/tree/master
-
 :uri-docs: {uri-rel-file-base}/docs
 
 :uri-oci: https://cloud.oracle.com/cloud-infrastructure
 :uri-oci-compartment: https://docs.cloud.oracle.com/iaas/Content/Identity/Tasks/managingcompartments.htm#two
 :uri-oci-ocids: https://docs.cloud.oracle.com/iaas/Content/API/Concepts/apisigningkey.htm#five
 :uri-oci-documentation: https://docs.cloud.oracle.com/iaas/Content/home.htm
 :uri-oci-keys: https://docs.cloud.oracle.com/iaas/Content/API/Concepts/apisigningkey.htm#two
-:uri-quickstart: {uri-docs}/quickstart.adoc
 :uri-terraform: https://www.terraform.io
 :uri-terraform-download: https://www.terraform.io/downloads.html
+:uri-hashicorp-brew-tap-announcement: https://www.hashicorp.com/blog/announcing-hashicorp-homebrew-tap
 
 This section will guide you through the pre-requisites before you can use this project.
 
-You can proceed to the {uri-quickstart}[Quick Start guide] if you have already done these.
+You can proceed to the xref:docs/quickstart.adoc[Quick Start guide] if you have already done these steps.
 
-. link:#install-terraform[Install Terraform]
-.. link:#install-terraform-on-oracle-linux[Install Terraform on Oracle Linux]
-.. link:#install-terraform-on-macos[Install Terraform on macOS]
-.. link:#manual-installation[Manual Installation]
-... link:#configure-path-on-linuxmacos[Configure path on Linux/macOS]
-... link:#configure-path-on-windows[Configure path on Windows]
-.. link:#testing-terraform-installation[Testing Terraform installation]
-. link:#generate-and-upload-your-oci-api-keys[Generate and upload your OCI API keys]
-. link:#create-an-oci-compartment[Create an OCI Compartment]
-. link:#obtain-the-necessary-ocids[Obtain the necessary OCIDs]
-. link:#identity-and-access-management-rights[Identity and Access Management Rights]
-. link:#using-autonomous-linux[Using Autonomous Linux]
+toc::[]
 
-=== Install Terraform
+== Install Terraform
 
 Start by installing Terraform and configuring your path.
 
-==== Installing Terraform on Oracle Linux
+=== Installing Terraform on Oracle Linux
```

Comment:
This would only work on Oracle Linux 7. 

For Oracle Linux 8, you'd need to run `dnf -y install oraclelinux-developer-release-el8 && dnf -y install terraform`